route/build/cpl/RtmMod.F90

```diff
@@ -638,12 +648,12 @@ SUBROUTINE route_run(rstwr)
         if (nRch_mainstem > 0) then
           lwr=1
           upr=nRch_mainstem
-          call get_river_export_data(NETOPO_main, RCHFLX_trib(:,lwr:upr))
+          call get_river_export_data(NETOPO_main(lwr:upr), RCHFLX_trib(:,lwr:upr))
```

Comment:
Do you really need to send an array section here? If it's really the whole array there isn't a need to send the section.

Reply:
The bound of NETOPO_main is 1:nRch_mainstem + nTribOutlet, and I needed to pass only 1:nRch_mainstem section for both `NETOPO_main` and `RCHFLX_trib`